src/main/java/codeu/controller/ConversationServlet.java

```diff
@@ -70,9 +70,15 @@ void setConversationStore(ConversationStore conversationStore) {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
+    long startTime = System.currentTimeMillis();
```

Comment:
I wonder if you could use Filters or something similar to them to write the profiling code once.

Something like this:
http://www.oracle.com/technetwork/tutorials/tutorials-1840472.html

Reply:
I will look into Filters for the next PR! I think we would probably be able to use them (at least for timing requests).